Content.Client/Lobby/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -693,13 +693,46 @@ public void RefreshAntags()
 
                 antagContainer.AddChild(selector);
 
-                antagContainer.AddChild(new Button()
+                // Moffstation - Begin - Enable loadouts for antags
+                var loadoutWindowBtn = new Button()
                 {
-                    Disabled = true,
+                    // Disabled = true,
                     Text = Loc.GetString("loadout-window"),
                     HorizontalAlignment = HAlignment.Right,
                     Margin = new Thickness(3f, 0f, 0f, 0f),
-                });
+                };
+
+                antagContainer.AddChild(loadoutWindowBtn);
+
+                var collection = IoCManager.Instance!;
+                var protoManager = collection.Resolve<IPrototypeManager>();
+
+                // If no loadout found then disabled button
+                if (!protoManager.TryIndex<RoleLoadoutPrototype>(LoadoutSystem.GetJobPrototype(antag.ID), out var roleLoadoutProto))
+                {
+                    loadoutWindowBtn.Disabled = true;
+                }
+                // else
+                else
+                {
+                    loadoutWindowBtn.OnPressed += args =>
+                    {
+                        RoleLoadout? loadout = null;
+
+                        // Clone so we don't modify the underlying loadout.
+                        Profile?.Loadouts.TryGetValue(LoadoutSystem.GetJobPrototype(antag.ID), out loadout);
+                        loadout = loadout?.Clone();
+
+                        if (loadout == null)
```

Comment:
```suggestion
                        RoleLoadout loadout;
                        if (Profile != null && Profile.Loadouts.TryGetValue(LoadoutSystem.GetJobPrototype(antag.ID), out var roleLoadout))
                        {
                            // Clone so we don't modify the underlying loadout.
                            loadout = roleLoadout.Clone();
                        }
                        else
```
imo this is more clear since there's less null coalescence stuff.